specs/002-issue-templates-align-sdd/plan.md

```
@@ -0,0 +1,278 @@
+    id: sdd_impact
+    attributes:
+      label: SDD Impact (Spec/Plan/Tasks Required?)
+      options:
+        - "Yes - requires full SDD workflow"
+        - "No - minor change only"
+        - "TBD - needs assessment"
+    validations:
+      required: true
+```
+
+### engineering_task.yml Complete Restructure
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 4; Actual: 2

(MD007, ul-indent)